src/renderer/renderElement.ts

```diff
@@ -811,6 +815,23 @@ const drawElementFromCanvas = (
       elementWithCanvas.canvas!.width / elementWithCanvas.canvasZoom,
       elementWithCanvas.canvas!.height / elementWithCanvas.canvasZoom,
     );
+
+    if (
+      process.env.REACT_APP_DEBUG_ENABLE_TEXT_CONTAINER_BOUNDING_BOX &&
+      hasBoundTextElement(element)
+    ) {
+      const coords = getContainerCoords(element);
+      context.save();
+      context.restore();
```

Comment:
```suggestion
```
Dead code, would be same even if not there, as we are not saving anything and just restoring it back. so..!